src/main/java/com/fundation/webservice/model/AudioResponse.java

```diff
@@ -21,63 +21,113 @@
-    private String aRate;
-
     private String checksum;
 
+    /**
+     * Method constructor  of the audio response class initial the all variable.
+     *
+     * @param fileName
+     * @param fileDownloadUri
+     * @param fileType
+     * @param size
+     * @param newFormat
+     * @param aCodec
+     * @param aBitRate
+     * @param aChannel
+     * @param checksum
+     */
     public AudioResponse(String fileName, String fileDownloadUri, String fileType, long size,
-            String newFormat, String aCodec, String aBit, String aChannel, String aRate, String checksum) {
+                         String newFormat, String aCodec, String aBitRate, String aChannel, String checksum) {
         this.fileName = fileName;
         this.fileDownloadUri = fileDownloadUri;
         this.fileType = fileType;
         this.size = size;
         this.newFormat = newFormat;
         this.aCodec = aCodec;
-        this.aBit = aBit;
+        this.aBitRate = aBitRate;
         this.aChannel = aChannel;
-        this.aRate = aRate;
         this.checksum = checksum;
     }
 
+    /**
+     * Method that return the name of outputFile
+     *
+     * @return name the output File.
+     */
     public String getFileName() {
         return fileName;
     }
 
+    /**
+     * Method that return URL of output file for Download.
+     *
+     * @return Uri the output File.
+     */
     public String getFileDownloadUri() {
         return fileDownloadUri;
     }
 
+    /**
+     * Method that return the Type of data of the output file.
+     *
+     * @return Type the output File.
+     */
     public String getFileType() {
         return fileType;
     }
 
+    /**
+     * Method that return the size of output File.
+     *
+     * @return Size the output File.
+     */
     public long getSize() {
         return size;
     }
 
+    /**
+     * Method that return the new format of the output File.
+     *
+     * @return new format of File
+     */
     public String getNewFormat() {
         return newFormat;
     }
 
+    /**
+     * Method that return the type the codec for convert the input File.
+     *
+     * @return codec for convert.
+     */
     public String getaCodec() {
         return aCodec;
     }
 
-    public String getaBit() {
-        return aBit;
+    /**
+     * Method that return bitrate for convert the input File
+     *
+     * @return bitrate the bitrate output file.
+     */
+    public String getaBitRate() {
+        return aBitRate;
     }
 
+    /**
+     * Method that return the number de channel for convert the input file.
+     *
+     * @return number the channel for output file.
+     */
     public String getaChannel() {
         return aChannel;
     }
 
-    public String getaRate() {
-        return aRate;
-    }
-
+    /**
+     * Method that return the checksum in format MD5 or SHA-1.
+     *
+     * @return checksum of the file.
+     */
     public String getChecksum() { return checksum; }
```

Comment:
if "return checksum" is only in one code line then you should convert all the "return" lines in one line code contrary case change this line like the other that contain "return".

Reply:
fix

Reply:
ok... that's good.